test/esjs.spec.js

```diff
@@ -316,6 +316,42 @@ describe('.search()', () => {
       });
     });
 
+    context('given two filters', () => {
+      const results = idx.search({
+        matchAll: {},
+        filter:   [
+          {
+            term: { category: 'crime' },
+          },
+          {
+            term: { status: 'PENDING_REVIEW' },
+          }],
+      });
+
+      it('returns the expected results', () => {
+        expect(searchIds(results)).to.eql([4]);
+      });
+    });
+
+    context('given query and two filters', () => {
+      const results = idx.search({
+        must: {
+          match: { title: 'sale' },
+        },
+        filter: [
+          {
+            term: { category: 'crime' },
+          },
+          {
+            term: { status: 'PENDING_REVIEW' },
+          }],
+      });
+
+      it('returns the expected results', () => {
```

Comment:
I'd prefer a positive match test, rather than a miss, here, just to make sure that the result is due to the exact mechanism we're testing.